NetExercise/Basic/Exercise10.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+
+namespace NetExercise.Basic
+{
+    public class Exercise10
+    {
+        public static float diff(float x, float n)
+        {
+            return x * x * x - n;
+        }
+
+        public static float CubeRoot(float n)
+        {
+            if (n < 0)
+            {
+                return -CubeRoot(-n);
+            }
+            else if (n == 0)
+            {
+                return 0;
+            }
+            else if (n == 1)
+            {
+                return 1;
+            }
+
+            float xl = 0;
```

Comment:
Sử dụng var.
Ví dụ

Khai báo số nguyên
```csharp
var x = 0;
```
Khai báo số thực float
```csharp
var y = 0f;
```
Khai báo số thực double
```csharp
var y = 0.0;
var y2 = 0D;
```

```suggestion
            var xl = 0.0; 
```